packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -246,6 +252,15 @@ export class FeatureDevController {
             case ZipFileError.errorName:
                 this.messenger.sendErrorMessage(errorMessage, message.tabID, 0, session?.conversationIdUnsafe, true)
                 break
+            case NoChangeRequiredException.errorName:
+                this.messenger.sendAnswer({
+                    type: 'answer',
+                    tabID: message.tabID,
+                    message: err.message,
+                    canBeVoted: true,
```

Comment:
nit - not sure about vote an empty patch. Is that something that will lead just to negative voting?

Reply:
We'd like users to provide feedbacks at any point of the experience.

Reply:
I think they can by the top level menu

<img width="552" alt="Screenshot 2024-10-04 at 12 55 50 PM" src="https://github.com/user-attachments/assets/26794d66-add9-4681-a69e-e6c3bb701617">

Reply:
Voting its more thumbs up/down, that can lead to the same screen.

Reply:
That one is for overall product feedback. We can not distinguish which feature users refers to. This is a requirement from product earlier.

Reply:
I see. So you mean voting we can? I cannot see how to "select" a feature here. That's what I see when thumbs down right now. Does this PR introduce a new behaviour related to that?

<img width="423" alt="Screenshot 2024-10-04 at 1 12 26 PM" src="https://github.com/user-attachments/assets/56c7af1c-223d-4bf8-92bc-9d26354033c3">

Reply:
User don't need select feature. It would contain the conversation context (e.g. feature, conversation) 
https://github.com/aws/aws-toolkit-vscode/commit/57382ad92d73f15591d119a28f5f2058d8a46fb2

Reply:
btw, overall feedback user should also be able to select which feature its talking about. I should be able to provide an overall feedback to feature dev anytime of the experience.

Reply:
Out of the PR topic. Discussed offline.